### PR TITLE
Add "RowsCopied" property to SqlBulkCopy

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Addresses #404

### CODE REVIEW

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs [hunk 1]
```diff
@@ -471,6 +471,15 @@ internal int ObjectID
 
         }
 
+        /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml' path='docs/members[@name="SqlBulkCopy"]/RowsCopied/*'/>
+        public int RowsCopied
```
Comment (Wraith2) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs]: Not a blocker just an observation, this could be expression form `public int RowsCopied => _rowsCopied;'

Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs]: I kept it same as other places to ensure it's readable enough that it's just the Getter that is available as public API.

Comment (Wraith2) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs]: Fair enough.

File: doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml [hunk 2]
```diff
@@ -471,6 +471,25 @@ and faster to use a Transact-SQL `INSERT … SELECT` statement to copy the data.
...
+			<value>
+				The integer value of the
+				<see cref="P:Microsoft.Data.SqlClient.SqlBulkCopy.RowsCopied" />
+				property.
+			</value>
+			<remarks>
+				<format type="text/markdown"><![CDATA[
+
+## Remarks
+This value is incremented during the <xref:Microsoft.Data.SqlClient.SqlBulkCopy.SqlRowsCopied> event and does not imply that these many rows have been sent to the server or committed.
```
Comment (David-Engel) [inline doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml]: I would rephrase this slightly:
... and does not imply that this number of rows has been sent to the server or committed.
